ios/RNTrackPlayer/Models/MediaURL.swift

```diff
@@ -12,19 +12,19 @@ struct MediaURL {
     let value: URL
     let isLocal: Bool
     private let originalObject: Any
-    
+
     init?(object: Any?) {
         guard let object = object else { return nil }
         originalObject = object
-        
+
         if let localObject = object as? [String: Any] {
             isLocal = true
             let uri = localObject["uri"] as! String
             let encodedURI = uri.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)!
             value = URL(string: encodedURI)!
         } else {
             let url = object as! String
-            isLocal = url.lowercased().hasPrefix("file://")
+            isLocal = url.lowercased().hasPrefix("file://") || url.lowercased().hasPrefix("http://localhost")
```

Comment:
What if someone was to use ‘https://localhost’?

Reply:
And how about `http://192.168.0.1` or `https://192.168.0.1`?